administrator-manual/en/webmail.rst

```diff
@@ -32,7 +32,7 @@ For example, given a server with IP address *192.168.1.1* and name *mail.mydomai
 
    .. note::
 
-       You can find a more recent version of roundcubemail in NethForge, you must remove first ``nethserver-roundcubemail`` and 
+       You can find a more recent version of roundcubemail in NethForge, you must remove first ``roundcubemail`` and 
```

Comment:
I would like to conflict `roundcubemail` and `nethserver-roundcubemail`, would it be possible ?

Reply:
in fact yum remove roundcubemail removes nethserver-roundcubemail and roundcubemail